browser/tor/tor_proxy_config_service.cc

```diff
@@ -27,9 +28,17 @@ const int kTorPasswordLength = 16;
 // Default tor circuit life time is 10 minutes
 constexpr base::TimeDelta kTenMins = base::TimeDelta::FromMinutes(10);
 
+const char kSocksProxy[] = "socks5";
+
+// This is a weird sorta-Windowsism meaning `don't bypass it for the
+// things that bypass it by default'.  For details, see
+// net/proxy_resolution/proxy_bypass_rules.cc.
+const char kTorProxyBypassRules[] = "<-loopback>";
```

Comment:
could we use the same name `kSubtractImplicitBypasses` which is more straightforward?

Reply:
It is private to proxy_bypass_rules.cc, so not without patching that file.

Reply:
just ignore this and use https://github.com/brave/brave-core/pull/2417/files#r283012179